activerecord/lib/active_record/autosave_association.rb

```diff
@@ -22,7 +22,7 @@ module ActiveRecord
   #
   # == Validation
   #
-  # Children records are validated unless <tt>:validate</tt> is +false+.
+  # Child records are validated unless <tt>:validate</tt> is +false+.
```

Comment:
Sorry for this, I'm not a native speaker but I think Child records are still plural and refers to all records belongs to parent, am I wrong? I've searched around source code to find same usage but I couldn't. There are only two usages like `Child records`. Nevermind maybe I've consumed lots of caffeine tonight, you are a native speaker and if you think this usage is correct just closing this pr :+1: